libs/scalalib/src/mill/scalalib/CoursierConfigModule.scala

```diff
@@ -0,0 +1,143 @@
+package mill.scalalib
+
+import coursier.CoursierEnv
+import coursier.cache.{CacheEnv, CachePolicy}
+import coursier.core.Repository
+import coursier.credentials.Credentials
+import coursier.params.Mirror
+import coursier.util.{EnvEntry, EnvValues}
+import mill.define.*
+import mill.{T, Task}
+
+import scala.concurrent.duration.Duration
+
+object CoursierConfigModule extends ExternalModule {
```

Comment:
Can you add some ScalaDoc explaining which entries we support and maybe a little example how they can be cutsomized?

We get asked for this every now and then, and we could answer: Just use `mill inspet mill.scalalib.CoursierConfigModule/`